Sources/OutlineView/OutlineView.swift

```diff
@@ -30,6 +44,109 @@ where Data.Element: Identifiable {
+
+    // MARK: NSViewControllerRepresentable
+    
+    public func makeNSViewController(context: Context) -> OutlineViewController<Data, Drop> {
+        let controller = OutlineViewController<Data, Drop>(
+            data: data,
+            childrenSource: childSource,
+            content: content,
+            selectionChanged: { selection = $0 },
+            separatorInsets: separatorInsets)
+        controller.setIndentation(to: indentation)
+        if #available(macOS 11.0, *) {
+            controller.setStyle(to: style)
+        }
+        return controller
+    }
+
+    public func updateNSViewController(
+        _ outlineController: OutlineViewController<Data, Drop>,
+        context: Context
+    ) {
+        outlineController.updateData(newValue: data)
+        outlineController.changeSelectedItem(to: selection)
+        outlineController.setRowSeparator(visibility: separatorVisibility)
+        outlineController.setRowSeparator(color: separatorColor)
+        outlineController.setDragSourceWriter(dragDataSource)
+        outlineController.setDropReceiver(dropReceiver)
+        outlineController.setAcceptedDragTypes(acceptedDropTypes)
+    }
+}
+
+// MARK: - Modifiers
+
+@available(macOS 10.15, *)
+public extension OutlineView {
+    /// Sets the style for the `OutlineView`.
+    @available(macOS 11.0, *)
+    func outlineViewStyle(_ style: NSOutlineView.Style) -> Self {
+        var mutableSelf = self
+        mutableSelf.style = style
+        return mutableSelf
+    }
+
+    /// Sets the width of the indentation per level for the `OutlineView`.
+    func outlineViewIndentation(_ width: CGFloat) -> Self {
+        var mutableSelf = self
+        mutableSelf.indentation = width
+        return mutableSelf
+    }
+
+    /// Sets the visibility of the separator between rows of this outline view.
+    func rowSeparator(_ visibility: SeparatorVisibility) -> Self {
+        var mutableSelf = self
+        mutableSelf.separatorVisibility = visibility
+        return mutableSelf
+    }
+
+    /// Sets the color of the separator between rows of this outline view.
+    /// The default color for the separator is `NSColor.separatorColor`.
+    func rowSeparatorColor(_ color: NSColor) -> Self {
+        var mutableSelf = self
+        mutableSelf.separatorColor = color
+        return mutableSelf
+    }
+
+    /// Adds a drop receiver to the OutlineView, allowing it to react to drag
+    /// and drop operations.
+    /// - Parameters
+    ///   - receiver: An object conforming to `DropReceiver` that handles
+    ///   - acceptedTypes: An array of `PasteboardType`s that the `DropReceiver` is able to read
+    ///   drag-and-drop receiving for the OutlineView
+    func onDrop(of acceptedTypes: [NSPasteboard.PasteboardType], receiver: Drop) -> Self {
+        var mutableSelf = self
+        mutableSelf.dropReceiver = receiver
+        mutableSelf.acceptedDropTypes = acceptedTypes
```

Comment:
I'm really nitpicking here, but can you set the variables in the same order as the parameters? 🙂

```suggestion +1-2
        mutableSelf.acceptedDropTypes = acceptedTypes
        mutableSelf.dropReceiver = receiver
```

Reply:
haha. No problem. Fixed here: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3